# Calculate State Trends of COVID-19 Cases for
Alaska, Alabama, Arkansas, Arizona, California, Colorado, Connecticut, Delaware, District of Columbia, and Florida. 

### Read Merged Covid-19 Dataset with Variable "State" as our index. 

In [1]:
import pandas as pd
import numpy as np

#read super Covid-19 dataframe with state as index. 
Read = pd.read_csv("../../../data/output/covid.csv", index_col="State")

### Group columns by index State, and add the cases of all counties within a state.

In [2]:
#groups the column by States, and adds up cases of each countie per state. 
States = Read.groupby(['State']).sum()


#Output the recent week. 
Trend = States[['9/8/20_x','9/9/20_x','9/10/20_x','9/11/20_x','9/12/20_x','9/13/20_x','9/14/20_x']]



### Outputs the first 10 states in alphabetical order. 

In [3]:
Trend[:10:]

9/8/20_x  9/9/20_x  9/10/20_x  9/11/20_x  9/12/20_x  9/13/20_x  \
State                                                                   
AK         5831      5894       6018       6110       6214       6276   
AL       133605    134416     135565     136703     137646     138753   
AR        66019     66402      66804      67911      68309      70217   
AZ       206044    206541     206993     207521     208126     208512   
CA       743148    747809     750062     755495     758504     760490   
CO        59671     59917      60183      60491      60904      61318   
CT        53782     53871      54093      54326      54326      54326   
DC        14362     14387      14412      14493      14552      14592   
DE        18308     18308      18466      18559      18726      18849   
FL       649979    652148     654731     658380     661569     663991   

       9/14/20_x  
State             
AK          6351  
AL        139454  
AR         70625  
AZ        208725  
CA        765475  
CO         61699  
CT         54895  
DC         14622  
DE         18937  
FL        665730

### Calculate Percentage Increase of each of the 10 states.

In [4]:
#Creates new column 'Percent' and calculates percentage increase for better anaylsis of trend.
States['Percent'] = ((States['9/14/20_x'] - States['9/8/20_x']) / States['9/8/20_x']) * 100
#Prints out first 10 States Trends of case numbers. 
MyState_Trend = States[['Percent']]
MyState_Trend[:10:]

Percent
State          
AK     8.917853
AL     4.377830
AR     6.976779
AZ     1.301178
CA     3.004381
CO     3.398636
CT     2.069466
DC     1.810333
DE     3.435657
FL     2.423309

# Analysis

## All 10 states COVID-19 cases have been increasing in the past recent week

#### Alaska's COVID-19 cases have increased by 8.9%
#### Alabama's COVID-19 cases have increased by 4.3%
#### Arkansas' COVID-19 cases have increased by 6.9%
#### Arizona's COVID-19 cases have increased by 1.3%
#### California COVID-19 cases have increased by 3.0%
#### Colorado COVID-19 cases have increased by 3.4%
#### Connecticut COVID-19 cases have increased by 2.0%
#### District of Columbia's COVID-19 cases have increased by 1.8%
#### Deleware COVID-19 cases have increased by 3.4%
#### Florida COVID-19 cases have increased by 2.42%